libs/shinkai-node-state/src/v2/queries/getJobFolderName/index.ts

```diff
@@ -10,5 +10,5 @@ export const getJobFolderName = async ({
   const result = await getJobFolderNameApi(nodeAddress, token, {
     job_id: jobId,
   });
-  return result;
+  return result.folder_name;
```

Comment:
this will break some places in the app where the useGetJobFolderName is used, I'd recommend reverting it and add the `[].folder_name` for the open chat folder